cmake/winml.cmake

```diff
@@ -337,6 +340,9 @@ add_dependencies(winml_lib_api winml_api_native_internal)
 
 # Link libraries
 target_link_libraries(winml_lib_api PRIVATE wil)
+if (onnxruntime_USE_DML)
```

Comment:
>onnxruntime_USE_DML [](start = 4, length = 19)

what was this one for?  this is all on teh github side, right?

Reply:
lib_api shouldn't need dml, right? it was just lib_api_image?

---
In reply to: [354635943](https://github.com/microsoft/onnxruntime/pull/2565#discussion_r354635943) [](ancestors = 354635943)